fem/intrules.hpp

```diff
@@ -293,7 +294,8 @@ class Quadrature1D
       GaussLobatto    = 1,
       OpenUniform     = 2,  ///< aka open Newton-Cotes
       ClosedUniform   = 3,  ///< aka closed Newton-Cotes
-      OpenHalfUniform = 4   ///< aka "open half" Newton-Cotes
+      OpenHalfUniform = 4,  ///< aka "open half" Newton-Cotes
+      ClosedGL        = 5   // Closed Gauss Legendre
```

Comment:
Convert to a doxygen comment and make comment inline with the style used in the above comments

Reply:
Will do so.